### PR TITLE
Take encryption params from Connection URI

### PR DESCRIPTION
**Pull request by MacondoExpress**
# Description

This pull request wants to remove the secure checkbox from the GraphQL Toolbox.
It now uses the protocol in the Connection URI to describe the behavior. If the encryption is not specified, then it is forced to be unencrypted to avoid the implicit redirect from the`neo4j-javascript-driver` when served on HTTPS sites.

Closes:
#1526

### CODE REVIEW

**Comment by neo4j-team-graphql** [comment]
Looks like you've updated the UI!

Check out your changes at
 http://neo4j-graphql-toolbox-stage-1530.surge.sh (=> for local development; use for instance [Neo4j Desktop](https://neo4j.com/download/))
or
 https://neo4j-graphql-toolbox-stage-1530.surge.sh (=> use to connect to an [Aura Free](https://neo4j.com/cloud/aura-free/) dbms)

File: packages/graphql-toolbox/src/contexts/auth.tsx [hunk 1]
```diff
@@ -88,8 +86,12 @@ export function AuthProvider(props: any) {
     [value, setValue] = useState<State>({
         login: async (options: LoginOptions) => {
             const auth = neo4j.auth.basic(options.username, options.password);
-            const encrypted = options.secure === "on" ? "ENCRYPTION_ON" : "ENCRYPTION_OFF";
-            const driver = neo4j.driver(options.url, auth, { encrypted });
+            const protocol = new URL(options.url).protocol;
+            // Manually set the encryption to off if it's not specified in the Connection URI to avoid implicit encryption in https domain
+            const driver = protocol.includes("+s") ?
```
[the remark this answers is not in the document]
Comment (tbwiss) [reply packages/graphql-toolbox/src/contexts/auth.tsx:91]: Connecting to a DBMS hosted on Neo4j Desktop that has the HTTPS connector enabled and is using a self-signed certificates works with the following protocols: `bolt`, `bolt+s`, `bolt+ssc`, `neo4j`, `neo4j+s`, `neo4j+ssc`
If a DBMS (on Neo4j Desktop) does not have the HTTPS connector enabled only the protocols `bolt` and `neo4j` are working.

**Review by tbwiss — APPROVED** [review]
LGTM!
Good fix @MacondoExpress!

File: packages/graphql-toolbox/src/contexts/auth.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (MacondoExpress) [reply packages/graphql-toolbox/src/contexts/auth.tsx:91]: Perfect, I merge it then!

**Pull request merged**